src/auteur-widget/auteur-widget.html

```diff
@@ -2,17 +2,21 @@
   <require from="./styles/auteur-widget.css"></require>
   <ux-dialog class="oe-dialog auteur-toevoegen-dialog">
     <ux-dialog-header>
-      <h3>${title}</h3>
+      <h3>${isEigenaarVermogensrecht ? 'Eigenaar vermogensrechten ' : 'Auteur '} toevoegen</h3>
     </ux-dialog-header>
 
     <ux-dialog-body>
       <div class="row">
-        <div class="large-12 columns">
+        <div class="large-8 columns">
           <div class="placeholder-container">
-            <input type="text" focus.one-way="true" placeholder="&#xF002; Zoek een auteur" value.bind="zoekterm"
+            <input type="text" focus.one-way="true" placeholder="&#xF002; Zoek een ${isEigenaarVermogensrecht ? 'eigenaar vermogensrechten' : 'auteur' } op naam of orcid" value.bind="zoekterm"
               keydown.delegate="search() & debounce:300" style="font-family: FontAwesome, 'Flanders Art Sans'">
           </div>
         </div>
+        <div class="large-4 columns">
+          <input type="checkbox" id="checkbox-erkende-collegas" change.delegate="search()" class="checkbox-custom" checked.bind="erkendeCollegas">
+          <label for="checkbox-erkende-collegas" class="checkbox-custom-label">Toon enkel mijn erkende collega's</label>
+        </div>
```

Comment:
was deze filter ook nuttig voor inventaris/beeldbank?

Reply:
gaan we dees dan gans schrappen

Reply:
Kheb er een if check rond gezet

Reply:
deze if mag wijzigen naar een check op de aanwezigheid van auteur organisaties.